blocks/pull-quote/pull-quote.css

```diff
@@ -0,0 +1,27 @@
+/* variables for pull-quote block */
+main .pull-quote {
+    --color-quote: #367ea8;
+}
+/**/
+main .pull-quote {
+    border-left: 3px solid var(--color-quote);
+    padding: 12px 24px;
+    margin: 64px 0;
+}
+main .pull-quote p {
+    margin: .5em 0;
+}
+main .pull-quote img {
+    height: 96px;
+    width: 96px;
+    object-fit: cover;
+    border-radius: 100%;
+    margin: 0;
+}
+main .pull-quote blockquote {
+    margin: 32px 0 0 0;
+    color: var(--color-quote);
+    font-weight: var(--heading-font-weight);
+    line-height: var(--heading-line-height);
+    font-size: var(--heading-font-size-l);
```

Comment:
@davidnuescheler I added this font update to make the blockquote looks like h2. It would be nice if I could just add a class .h2 instead but I don't see that was provided by styles.css.
I also found that h1 is only has main in front of it while h2 to h6 doesn't. I wonder if this was just a mistake from last PR https://github.com/adobe/business-website/pull/4 . I think this should be main h1, main h2, main h3 ...
![image](https://user-images.githubusercontent.com/55804872/126563036-3509586b-9231-401f-8ebe-aca5b7df5aa0.png)

Reply:
yeah, i think that's a bug. i will fix that in `styles.css`...

i think you bring a up a good point that it would be great to have css classes that bring together the formatting of a the `xxl`, `xl`, `l` etc. headings... it is important though to mention that the semantic headings eg. `<h1>`, `<h2>` should not be tied verbatim to a style via a class name that implies semantics. but we should have this conversation in a separate issue probably as this is something that we should agree upon across the project.

cc: @fkakatie

Reply:
Thanks! I will look forward to have some utility classes for common styles.

Reply:
Thanks! I will look forward to have some utility classes for common styles.